consensus/polybft/polybft.go

```diff
@@ -131,7 +131,7 @@ func GenesisPostHookFactory(config *chain.Chain, engineName string) func(txn *st
 		}
 
 		if err = initContract(contracts.ValidatorSetContract, input, "ChildValidatorSet", transition); err != nil {
-			return err
+			return fmt.Errorf("ChildValidatorSet SC call failed! error: %w", err)
```

Comment:
Reworked the docker scripts, so that these code changes are not needed. 
Reverted all the code changes, so now only docker related files are changed.